### PR TITLE
feat(FEC-13598): update color tokens

### PR DESCRIPTION
**Pull request by lianbenjamin**
### Description of the Changes

update color tokens per design.

#### Resolves [FEC-13598](https://kaltura.atlassian.net/browse/FEC-13598)

[FEC-13598]: https://kaltura.atlassian.net/browse/FEC-13598?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/styles/dynamaic-variables.scss [hunk 1]
```diff
@@ -85,6 +85,6 @@
 
   --playkit-background-color: #111111;
```
Comment (SivanA-Kaltura) [inline src/styles/dynamaic-variables.scss:86]:
```suggestion
  --playkit-background-color: #121212;
```

File: src/styles/dynamaic-variables.scss [hunk 2]
```diff
@@ -70,10 +70,10 @@
...
-  --playkit-tone-5-color: #666666;
-  --playkit-tone-6-color: #444444;
-  --playkit-tone-7-color: #222222;
+  --playkit-tone-4-color: #666666;
+  --playkit-tone-5-color: #525252;
+  --playkit-tone-6-color: #383838;
+  --playkit-tone-7-color: #212121;
   --playkit-tone-8-color: #000000;
 
   --playkit-live-color: #e12536;
```
Comment (SivanA-Kaltura) [inline src/styles/dynamaic-variables.scss:79]:
```suggestion
  --playkit-live-color: #e12437;
```

File: src/styles/dynamaic-variables.scss [hunk 3]
```diff
@@ -70,10 +70,10 @@
   --playkit-tone-1-color: #ffffff;
```
Comment (SivanA-Kaltura) [inline src/styles/dynamaic-variables.scss:70]: @lianbenjamin Check the values that are defined by hue-saturation. It looks like their values were also updated.

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

File: src/styles/dynamaic-variables.scss (hunk 3 above)
Comment (lianbenjamin) [reply src/styles/dynamaic-variables.scss:70]: @SivanA-Kaltura i've checked them with an online hsl to hex tool (and vice-versa)- they match the design.

**Review by SivanA-Kaltura — APPROVED** [review]

**Pull request merged**